src/server/routes/magicLink.ts

```diff
@@ -10,17 +10,23 @@ import { ResponseWithRequestState } from '@/server/models/Express';
 import { handleAsyncErrors } from '@/server/lib/expressWrappers';
 import { ApiError } from '@/server/models/Error';
 import handleRecaptcha from '@/server/lib/recaptcha';
+import { rateLimiterMiddleware } from '../lib/middleware/rateLimit';
 
-router.get('/magic-link', (req: Request, res: ResponseWithRequestState) => {
-  const html = renderer('/magic-link', {
-    requestState: res.locals,
-    pageTitle: 'Sign in',
-  });
-  res.type('html').send(html);
-});
+router.get(
+  '/magic-link',
+  rateLimiterMiddleware,
+  (req: Request, res: ResponseWithRequestState) => {
+    const html = renderer('/magic-link', {
+      requestState: res.locals,
+      pageTitle: 'Sign in',
+    });
+    res.type('html').send(html);
+  },
+);
```

Comment:
I think there is a discussion to be had about whether it's worth rate limiting simple GET requests that aren't going to do anything other than render HTML. If the request isn't going to in turn spawn requests to other services then it may be that the 10ms or however long it may be waiting for a response from redis is not worth it?

Reply:
just documenting our chat outside of the PR,  — we've been told by infosec that it is good practice to apply the rate limiter to all route handlers to degrade the ability of bad actors to find a vulnerable attack surface. what we can do though is extend the private configuration to disable the rate limiter for specific routes if we do decide that it isn't needed